packages/allure-playwright/src/index.ts

```diff
@@ -143,7 +150,12 @@ class AllureReporter implements Reporter {
     if (!this.options.detail && step.category !== "test.step") {
       return;
     }
-    this.ensureAllureStepCreated(step, allureTest);
+    const allureStep = this.ensureAllureStepCreated(step, allureTest);
```

Comment:
maybe we can better to skip creating the step itself for the stepAttachments? Because we can add attachments directly to the `step.parent``